### PR TITLE
changed links to mapquest terms

### LINKED ISSUE #4401: Links to MapQuest terms broken in LICENSE.md
**Describe the bug**
`LICENSE.md` contains http://developer.mapquest.com/web/info/terms-of-use and http://info.mapquest.com/terms-of-use/ as links to the MapQuest terms:
- the first is redirected to the homepage
- the last returns 403

**Where was the bug observed**
`LICENSE.md`, both server and client

**Expected behavior**
Well, find information about MapQuest terms :smirk:

### PR DESCRIPTION
**Pull request by andrew21-mch**
This pull request makes the following changes:
- changes links to the mapquest terms
- tested link on chrome browser

- [x] I certify that I ran my checklist

Fixes #4401 

Ping @ushahidi/platform

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -40,4 +40,4 @@ Map Data
 ========
 
 The default map data used comes from OpenStreetMap - <http://osm.org/copyright>
-The default imagery comes from MapQuest and is subject to the following terms: <http://developer.mapquest.com/web/info/terms-of-use> and <http://info.mapquest.com/terms-of-use/>
+The default imagery comes from MapQuest and is subject to the following terms: <https://developer.mapquest.com/legal> and <https://hello.mapquest.com/terms-of-use>
```
Comment (ptandler) [inline LICENSE.md:43]: Would be helpful to explain why there are _two_ links (not just one) and what they govern in detail ... in case somebody knows ...?

Comment (andrew21-mch) [reply LICENSE.md:43]: hi @ptandler  guess I am still trying to figure it out, 
- I noticed one governs the developers ( https://developer.mapquest.com/legal ) and the other governs general terms of use (https://hello.mapquest.com/terms-of-use ) , its still not really that clear to me yet.

**Comment by andrew21-mch** [comment]
hi @webong please whats you take on this?

**Comment by ptandler** [comment]
@Angamanga - this might be a kind of stupid question: I just realized, that Ushahidi seems to use Mapbox.com, not MapQuest ... so if this is correct, should the license mention MapBox instead?

When I do a full-text search in the code, I only find this usage of "mapquest":

```js
            if (config.default_view.baselayer === 'MapQuest') {
                config.default_view.baselayer = 'streets';
            }
            if (config.default_view.baselayer === 'MapQuestAerial') {
                config.default_view.baselayer = 'satellite';
            }
```

However, the mapbox API key is mentioned in a couple of places: `apikey: CONST.MAPBOX_API_KEY,`

**Comment by andrew21-mch** [comment]
Hi @ptandler thanks,
please what should be done next in this case ?

**Comment by andrew21-mch** [comment]
> > Hi @ptandler thanks, please what should be done next in this case ?
> 
> Hm, I think this is mainly a legal issue, so _someone_ who understands legal stuff would need to clarify the situation here. I am a _developer_ sunglasses

Alright thanks

**Pull request closed without merging**